src/frontend/test-setup.ts

```diff
@@ -6,6 +6,20 @@ jest.mock("./src/environment.ts", () => ({
   BASE_URL: "/",
 }));
 
+jest.mock("./src/features.ts", () => ({
+  FETCH_ROOT_KEY: false,
+  DUMMY_AUTH: false,
+  DUMMY_CAPTCHA: false,
+}));
+
+jest.mock("./src/version.ts", () => ({
+  version: {
+    commit: "",
+    release: undefined,
+    dirty: false,
+  },
+}));
```

Comment:
shouldn't all env variables be read in `environment.ts`?

Reply:
See PR description.

Reply:
Do you mean this?

> Refactoring: Integrate version.ts into environment.ts

I wasn't quite sure what it was referring to. Will this include `feature.ts` too?

Reply:
I was not planing to move `features.ts`, thought you like it that way. No strong opinion, we can move these as well, your call.

Reply:
Ah sorry, I think we can keep `features.ts` but would make sense to import the environment values from `environment.ts`. Otherwise we'd be pulling in env vars from two different places (and it would break if any test imports `features.ts` (transitively))